doc/whatsnew/fragments/5362.feature

```diff
@@ -0,0 +1,3 @@
+Added the ``no-header`` output format. If enabled, it will not include the module name in the output.
```

Comment:
```suggestion
Added the ``no-header`` output format. If enabled with ``--output-format=no-header``, it will not include the module name in the output.
```